falcon/request.py

```diff
@@ -358,10 +358,23 @@ class Request(object):
             accessed.)
         range_unit (str): Unit of the range parsed from the value of the
             Range header, or ``None`` if the header is missing
-        if_match (str): Value of the If-Match header, or ``None`` if the
-            header is missing.
-        if_none_match (str): Value of the If-None-Match header, or ``None``
-            if the header is missing.
+        if_match (list): A list containing all the etags in the If-Match
+            header.
+
+            This property is determined by the value of ``HTTP_IF_MATCH``
+            in the WSGI environment dict.
+
+            Note:
+                This property includes strong and weakness entity-tags and that
```

Comment:
If this sentence is rephrased from the relevant RFC, could we also consider linking to that RFC?

Furthermore, I would start a new sentence instead of _and that_, and would cite the RFC there.